### ISSUE TITLE
Constructor without documentation should show title of class

### ISSUE THREAD

**Issue by robstoll** [Kotlin/dokka#189]
Following an example:
```
/**
 * Represents xyz bla bla bla ....
 *
 * Some additional information
 *
 * @param T type of ...
 *
 * @constructor Represents xyz bla bla bla ....
 * @param value Some doc for value ....
 */
class A<T>(val value: Int){}
```
Most of the time I want that the same sentence for the constructor appears as I wrote as title of the class. Right now I have to duplicate what I have written which a) is cumbersome b) a very bad for maintenance of the documentation.

It would be nice, if the title (the first sentence) of the class would be used if no description is given after @cosntructor

I am willing to help out coding this feature, just let me know how to start.

**Comment by yole** [comment]
Why do you need the `@constructor` tag at all in this example? You can use `@param` directly in the class documentation to document primary constructor parameters.

**Comment by robstoll** [comment]
If I leave out @constructor then intellij interprets `@param value` as type parameter (I guess) and does not show it when I do ALT + Mouse-middle-click on the constructor. It would show the documentation though. However, if I want to have both, param and constructor description then I have to do it like in the example

**Comment by kamilok1965** [comment]
Hi @robstoll, I think this was fixed in IntelliJ eventually, didn't it?

**Comment by robstoll** [comment]
Seems to be fixed in intellij -- might be it always was, this is about a bug in dokka. And there it still doesn't work as I would like (at least with dokka 0.9.18). You can try it out with the above example. The bad thing, it seems to be now even worse than it was. You don't have to remove the doc from `@constructor` (which was the case when I wrote this bug back in 2017), now it will never print a documentation for the constructor, an empty doc respectively. At least the documentation for the param is still there.
Does it work for you?

**Comment by Kordyjan** [comment]
As I understand it is fixed in 1.4.0. If it is not, feel free to reopen.
